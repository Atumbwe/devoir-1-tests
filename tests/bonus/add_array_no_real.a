5 2 -5 -7 complex +