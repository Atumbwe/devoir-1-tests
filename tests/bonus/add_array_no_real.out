 _____
< -5i >
 -----
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||
